pkg/synchronization/controller.go

```diff
@@ -72,6 +75,10 @@ type ServiceMetadataStore interface {
 	ListModifiedServices(ctx context.Context, user auth.OptionalUser, modifiedSince time.Time) ([]creator_v1.Service, error)
 }
 
+type OpsGenieIntegrationManagerClient interface {
+	GetOrCreateIntegrations(ctx context.Context, teamName string) (bool, *opsgenieIntegrationManager.IntegrationsResponse, error)
```

Comment:
please swap first and second return values

Reply:
also add comment for what bool means please